test/e2e/feature/feature.go

```diff
@@ -269,7 +269,8 @@ var (
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
 	MasterUpgrade = framework.WithFeature(framework.ValidFeatures.Add("MasterUpgrade"))
 
-	// TODO: document the feature (owning SIG, when to use this feature for a test)
+	// owner: sig-node
+	// It is used for jobs in test-infra to only run memory manager tests.
```

Comment:
this is a bad explanaition. The feature must only be used to request a special environment. I would assume it asks for multi-NUMA nodes?

/cc @mariafromano-25